### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

**Review by abdulflorez — APPROVED** [review]

**Review by generaciondesafiante — APPROVED** [review]

File: middlewares/confirm-mail-changePassword.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    if (!user) {
+      throw new Error("Usuario no encontrado");
+    }
+
+    await transporter.sendMail({
+      from: user.email,
+      to: email,
+      subject: "Confirmación de Cambio de Contraseña",
+      html: `
+        <div style="font-family: Arial, sans-serif; max-width: 600px; margin: 0 auto; padding: 20px; border: 1px solid #dcdcdc; border-radius: 10px;">
```
Comment (DanielCalleCO) [inline middlewares/confirm-mail-changePassword.js:24]: Question... these styles should be in px?

Comment (Daniielmayo) [reply middlewares/confirm-mail-changePassword.js:24]: Yes, it should go in px, but for now I'm going to consider it as a technical debt to be able to merge it to main

File: controllers/courses/addResourcesToTopic.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        .status(400)
+        .send({ error: "El recurso ya está agregado a este tema" });
+    }
+
+    // if (course.typeOfRoute === "strict" && typeof isMandatory !== "boolean") {
+    //   return res.status(400).send({
+    //     error:
+    //       "Para cursos flexibles, el valor de isMandatory debe ser obligatorio",
+    //   });
+    // }
```
Comment (DanielCalleCO) [inline controllers/courses/addResourcesToTopic.js:38]: it's neccessary to keep these commented lines?

File: controllers/user/lastViewedResourceUser.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+const mongoose = require("mongoose");
+const User = require("../../models/User");
+const Course = require("../../models/Courses");
+const Resource = require("../../models/Resources");
+
+const lastViewedResource = async (req, res = response) => {
+  const { userId, courseId, topicId, resourceId } = req.body;
+
+  try {
+    // Validar si el userId es válido
```
Comment (DanielCalleCO) [inline controllers/user/lastViewedResourceUser.js:11]: Could you please translate these comments to english?

**Review by DanielCalleCO — APPROVED** [review]

**Pull request merged**